assets/main.scss

```diff
@@ -62,3 +62,124 @@ $on-laptop:        800px;
 a.site-title:hover{
   text-decoration: none;
 }
+
```

Comment:
You don't need to rewrite the same rules again and again for different CSS classes. You can use something like:

´´´
.one-class, .another-class {
  height: 2rem;
  // whatever rules you need
}
´´´

Reply:
I admit that my older commits on NLP were much more descriptive. The front-end world is completely new for me. You're right on the CSS code - I knew you would refactor it : )Thanks a lot. No more blog commits for now.

Reply:
Sure, no problem at all :) just a bit of feedback here and there. You've done a great work with the blog. Let me know when you want to make it available online again.